src/commands/contract/index.ts

```diff
@@ -201,6 +203,10 @@ function runPrettier(codeText: string) {
         trailingComma: 'es5',
         parser: 'typescript',
     })
+
+    const linter = new eslint.ESLint({useEslintrc: true, fix: true})
```

Comment:
I don't know if this is going to work outside of the repo since .eslintrc won't be defined. Why do you even need it though? What are the eslint checks that are failing without it?

Reply:
Oh was it this one? - https://github.com/wharfkit/cli/actions/runs/7015768034/job/19085669798

Reply:
If so then maybe you can just have your code fix that without needing eslint. Either that or you need to pass all the eslint configs here (instead of using the .eslintrc file).

Reply:
Yeah - we never managed to get eslint working when its not in dev mode, so we dropped it a couple versions ago.